examples/plot_importance_classification_iris.py

```diff
@@ -116,10 +118,16 @@ def run_one_fold(X, y, model, train_index, test_index, vim_name="CFI", groups=No
     GridSearchCV(SVC(kernel="rbf"), {"C": np.logspace(-3, 3, 10)}),
 ]
 cv = KFold(n_splits=5, shuffle=True, random_state=0)
-groups = {ft: [i] for i, ft in enumerate(dataset.feature_names)}
+features_groups = {ft: [i] for i, ft in enumerate(dataset.feature_names)}
```

Comment:
In that case,  features groups are trivial. We are making the example harder to understand for no good reason.

Reply:
I see, but it has to go.